platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -334,7 +334,11 @@ export default class HangingProtocolService extends PubSubService {
    *        the studies to display in viewports.
    * @param protocol is a specific protocol to apply.
    */
-  public run({ studies, displaySets, activeStudy }, protocolId) {
+  public run(
+    { studies, displaySets, activeStudy },
+    protocolId,
+    { stageIndex }
```

Comment:
Can you pass all the options here rather than extracting stageIndex?  There are reasons it might be desirable to pass initial display conditions to the run method.